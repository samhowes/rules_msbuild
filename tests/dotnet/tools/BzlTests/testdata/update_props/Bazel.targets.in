content that will be erased