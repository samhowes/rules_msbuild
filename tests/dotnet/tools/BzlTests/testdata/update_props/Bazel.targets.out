FAKE_BAZEL_TARGETS